lib/cobra_commander/component_tree.rb

```diff
@@ -93,6 +97,14 @@ def gemfile_path
         def gemfile_lock_path
           File.join(@root_path, "Gemfile.lock")
         end
+
+        def bundler_version_supporting_path_method?
+          @bundler_version_supporting_path_method ||= bundler_minor_version >= 16
+        end
+
+        def bundler_minor_version
+          bundler_definition.locked_bundler_version.split(".")[1].to_i
```

Comment:
Do you care about the locked version, or the version in use at runtime?

Reply:
The one in use at runtime.

Good catch with that method name. 

It appears though, that this actually returns the runtime version. As the locked version in this repo was `1.16.0` for me, but my local version was `1.16.3`.

---

Given how much Bundler's api appears to change, what about something like:

```ruby
stout, _ = Open3.capture2("bundler", "-v")
```

More reliable, or hacky?

Reply:
Less reliable: the version in use and the version in the path might potentially be different.

Is Bundler::VERSION not defined? If not, I would ask rubygems for the version of Bundler that has been loaded.

Reply:
Bundler::VERSION is accessible :).